auth/auth.go

```diff
@@ -25,14 +25,17 @@ import (
 )
 
 const (
-	CSRFCookieName   = "csrf-token"
-	CSRFHeader       = "X-CSRFToken"
-	errorOAuth       = "oauth_error"
-	errorLoginState  = "login_state_error"
-	errorCookie      = "cookie_error"
-	errorInternal    = "internal_error"
-	errorMissingCode = "missing_code"
-	errorInvalidCode = "invalid_code"
+	CSRFCookieName    = "csrf-token"
+	CSRFHeader        = "X-CSRFToken"
+	stateCookieName   = "state-token"
+	errorOAuth        = "oauth_error"
+	errorLoginState   = "login_state_error"
+	errorCookie       = "cookie_error"
+	errorInternal     = "internal_error"
+	errorMissingCode  = "missing_code"
+	errorMissingState = "missing_state"
+	errorInvalidCode  = "invalid_code"
+	errorInvalidState = "invalid_state"
```

Comment:
These new error messages new associated frontend descriptions here: https://github.com/openshift/console/blob/417be1e1587415f9c364bb26090e744e24d14938/frontend/public/components/error.jsx

Reply:
@Sean-Q-Sun still working on this bit ^ ?

Reply:
Just added!